maquetado/scss/_titles.scss

```diff
@@ -0,0 +1,19 @@
+.title {
+  align-items: center;
+  color: $black;
+  display: flex;
+  font-size: 24px;
+  height: $title-decoration-height;
+  position: relative;
+  width: 100%;
+}
+
+.title:before {
```

Comment:
Since we are using SASS, you can write less code by using the nesting feature:

```scss
.title {
   ...
   
   &:before { // the '&' operator will do the reference to it's parent in the compiled css, resulting in '.title:before'
    ...
   }
}
```